crates/sui-core/src/authority/authority_store.rs

```diff
@@ -777,19 +737,24 @@ impl AuthorityStore {
                 [(effects_digest, effects.data())],
             )?;
 
-        batch.write()?;
+        // Commit.
+        write_batch.write()?;
+
+        // Wait for results to become available.
+        // TODO: is this necessary? Would effects_nofity_exists() be better?
```

Comment:
i don't understand this comment - we aren't waiting for anything, we're notifying other waiters.

Reply:
Ok, mis-read this. Removed.